TYPO3.Fluid/Classes/TYPO3/Fluid/Core/ViewHelper/AbstractConditionViewHelper.php

```diff
@@ -113,6 +130,29 @@ protected function renderThenChild()
     }
 
     /**
+     * Statically evalute "then" children.
+     * The "$hasEvaluated" argument is there to distinguish the case that "then" returned NULL or was not evaluated.
+     *
+     * @param array $arguments ViewHelper arguments
+     * @param bool $hasEvaluated Can be used to check if the "then" child was actually evaluated by this method.
+     * @return string
+     */
+    protected static function renderStaticThenChild($arguments, &$hasEvaluated)
+    {
+        if (isset($arguments['then'])) {
```

Comment:
Nitpick: The $hasEvaluated should probably be initialized to true, else it might be error prone if f.e. the caller forgot to initialize the variable to true itself beforehand.